.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -14,7 +14,13 @@ Baritone version:
 Forge mods (if used): 
 
 ## Exception, error or logs
-You can find your logs in `%appdata%/.minecraft/logs/` (Windows) or `/Library/Application\ Support/minecraft/logs` (Mac).
+Please find your `latest.log` or `debug.log` in this folder and attach it to the issue
+
+Linux: `~/.minecraft/logs/`
+
+Windows: `%appdata%/.minecraft/logs/`
+
+Mac:`/Library/Application\ Support/minecraft/logs/`
```

Comment:
s p a c e

Reply:
what space alksdlksalk

the lines? you'd need to make it all one line when rendering without it